qa/tasks/cephadm.py

```diff
@@ -465,10 +465,6 @@ def ceph_bootstrap(ctx, config):
         ])
 
         # set options
-        if config.get('allow_ptrace', True):
-            _shell(ctx, cluster_name, bootstrap_remote,
-                   ['ceph', 'config', 'set', 'mgr', 'mgr/cephadm/allow_ptrace', 'true'])
```

Comment:
so the question is: do we need to set the container-cli-args here? I think we still need the to add the correspoinding  arguments to podman at this point